package.json

```diff
@@ -19,13 +19,14 @@
     "test": "jest"
   },
   "engines": {
-    "node": ">=10"
+    "node": "^14.0.0",
+    "npm": "^6.0.0"
```

Comment:
This will be a warning for everyone using npm 7 🤔 do we want that?

Reply:
I'm okay with that, since npm 7 is still a little buggy afaik

Reply:
Updated in #51

Reply:
I don't think this tool should be restricted to use with `npm@6` thought. What's the benefit in that?

Reply:
We updated it to `>=6` in #50. Do you think we should support lower versions of npm?

Reply:
No, that's fine, missed the other change 🙇‍♂️